config/myCon.json

```diff
@@ -0,0 +1,18 @@
+{
+	"mlist":[
```

Comment:
Don't worry about pushing small configs to robots in display. That might be considered when we talk about adding robots during runtime. For now, that will be handled by the deployment script of dockerize-srhmd.